### DISCUSSION TITLE
assertEq error during FundMe test build

### DISCUSSION THREAD

**Discussion by thecaptncrunch** [Cyfrin/foundry-full-course-cu#2651] [Q&A] [answered]
<img width="1454" alt="Screenshot 2024-10-09 at 7 07 16 PM" src="https://github.com/user-attachments/assets/867b1e66-76ca-46ef-b482-fd0c3986498c">

Compilation of FundMeTest.t.sol is failing due to an error `Error (7576): Undeclared identifier. Did you mean "assert"?` reviewing with chatgpt states to make sure the right packages are installed but I have the latest Foundry packages. Please advise!

**Comment by BowtiedHarpyEagle** [comment]
Try changing the line to 
assert(fundMe.MINIMUM_USD() == 5e18);

**Comment by EngrPips** [comment, marked as answer]
Hello @thecaptncrunch, Your test contract is supposed to look like the one below.

```js
contract FundMeTest is Test {

}
```
If you don't declare your test contract as a Test type, you won't have access to the `assert`, `assertEq`, and every other test superpower.

**Reply by thecaptncrunch** [reply]
Thank you! This worked like a charm

**Reply by EngrPips** [reply]
You are welcome, Friend. Keep crushing it.
